app/services/intake_progress_calculator.rb

```diff
@@ -1,8 +1,11 @@
 class IntakeProgressCalculator
 
   starting_index = QuestionNavigation::FLOW.index(Questions::LifeSituationsController)
-  ending_index = QuestionNavigation::FLOW.index(Questions::AdditionalInfoController)
-  POSSIBLE_STEPS = QuestionNavigation::FLOW[starting_index..ending_index]
+  ending_index = QuestionNavigation::FLOW.index(Questions::SuccessfullySubmittedController)
+  question_steps = QuestionNavigation::FLOW[starting_index..ending_index]
+  doc_overview_index = question_steps.index(Questions::OverviewDocumentsController)
+
+  POSSIBLE_STEPS = question_steps.insert(doc_overview_index + 1, *DocumentNavigation::FLOW)
```

Comment:
this inserts the documents steps into the set of possible steps at the right position.